components/TeamLeaderboard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Image from 'next/image';
+
+export default function TeamLeaderboard() {
+
+    const data = [
+        {
+            rank: 1,
+            talentScore: 1016.79,
+            name: "Alabama",
+            image: "https://a.espncdn.com/i/teamlogos/ncaa/500/333.png",
+            conference: "SEC"
+        },
+        {
+            rank: 2,
+            talentScore: 989.76,
+            name: "Georgia",
+            image: "https://a.espncdn.com/i/teamlogos/ncaa/500/61.png",
+            conference: "SEC"
+        },
+        {
+            rank: 3,
+            talentScore: 983.14,
+            name: "Ohio State",
+            image: "https://a.espncdn.com/i/teamlogos/ncaa/500/194.png",
+            conference: "Big Ten"
+        }
+    ]
+
+    return (
+        <div>
+            <>            
+                <div className="grid grid-cols-5 ml-28 mr-28 rounded-md text-xl mt-10 font-bold text-gray-200">  
+                    <h1 className="col-start-2">Rank</h1>                
+                    <h1>Team</h1>                
+                    <h1>Talent Score</h1>                
+                    <h1>Conference</h1>            
+                </div>           
+                <ul>
+                    {data.map((team) => (
```

Comment:
This is the error from vercel 
<html>
<body>
<!--StartFragment-->

./components/TeamLeaderboard.js
--
10:56:48.461 | 41:25  Error: Missing "key" prop for element in iterator  react/jsx-key
10:56:48.461 | 42:66  Warning: Image elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text

<!--EndFragment-->
</body>
</html>